index.html

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#">
     <head>
         <meta http-equiv="content-type" content="text/html; charset=UTF8" />
-        <meta name="description" content="Electricity Map | Real-time visualisation of the climate footprint of electricity generation" />
+        <meta name="description" content="Electricity Map | Clean vs. dirty electricity production in Europe" />
```

Comment:
OK makes sense. About real-time, I'd add one important aspect, which is that with wind and solar, I'd like to emphasise that the real-time component is important because of the fluctuating availability. Energy systems are conventionally thought to be "static", so the distinction was important.

What about `Real-time climate impact of the European electricity production`?